server/src/gateways/judges.gateway.ts

```diff
@@ -16,10 +16,11 @@ import { FightEndConditionFulfilledResponse } from '../interfaces/fight-end-cond
 import { FightEndConditionName } from '../interfaces/fight.interface';
 import { FightImpl } from '../classes/fight.class';
 import { JoinResponse } from '../interfaces/join-response.interface';
+import { PlayersService } from '../services/players.service';
 
 @WebSocketGateway()
 export class JudgesGateway implements FightEndConditionFulfilledObserver {
-  constructor(private fightsService: FightsService) {
+  constructor(private fightsService: FightsService, private playersService: PlayersService) {
```

Comment:
Dobrze byłoby przed wysłaniem commita użyć ESLint. Nie jest to duży problem przy sprawdzaniu kodu, gdy jest czytelny, jednak jeśli kolejna osoba używa ESLinta, to następne commity są po brzegi wypełnione tymi samymi, ale sformatowanymi linijkami. Dlatego najlepiej jest ujednolicić styl kodu. Jeśli używasz IDE, to wystarczy odpowiedni skrót klawiszowy, a jeśli nie, to możesz odpalić z konsoli `npm run lint`.